### DISCUSSION TITLE
How do I truncate the absolute file path in tags?

### DISCUSSION THREAD

**Discussion by idelice** [cbochs/grapple.nvim#115] [General]
As the title says, I have very long paths for tagged files and I would like to truncate them such that I only see the file name along with its extension e.g. foo.java.

How do I accomplish this?

**Comment by cbochs** [comment]
I am going to assume you found the setting for it, but if not, you set the `style` in the [settings](https://github.com/cbochs/grapple.nvim?tab=readme-ov-file#settings):

```lua
    ---How a tag's path should be rendered in Grapple windows
    ---  "relative": show tag path relative to the scope's resolved path
    ---  "basename": show tag path basename and directory hint
    ---@type "basename" | "relative"
    style = "relative",
```
